packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/loadTestAdministration/LoadTestAdministrationClient.ts

```diff
@@ -57,119 +27,11 @@ export class LoadTestAdministrationClient {
   ) {
     this._client = createLoadTestAdministration(endpoint, credential, options);
     this.pipeline = this._client.pipeline;
-  }
-
-  /** Create a new test or update an existing test. */
-  createOrUpdateTest(
-    testId: string,
-    body: Test,
-    options: CreateOrUpdateTestOptions = { requestOptions: {} }
-  ): Promise<Test> {
-    return createOrUpdateTest(this._client, testId, body, options);
-  }
-
-  /** Associate an app component (collection of azure resources) to a test */
-  createOrUpdateAppComponents(
-    testId: string,
-    body: TestAppComponents,
-    options: CreateOrUpdateAppComponentsOptions = { requestOptions: {} }
-  ): Promise<TestAppComponents> {
-    return createOrUpdateAppComponents(this._client, testId, body, options);
-  }
-
-  /** Configure server metrics for a test */
-  createOrUpdateServerMetricsConfig(
-    testId: string,
-    body: TestServerMetricConfig,
-    options: CreateOrUpdateServerMetricsConfigOptions = { requestOptions: {} }
-  ): Promise<TestServerMetricConfig> {
-    return createOrUpdateServerMetricsConfig(
-      this._client,
-      testId,
-      body,
-      options
+    this.administrationOperations = getAdministrationOperationsOperations(
```

Comment:
In this case, there's no `@operationGroup` decorator in the the client.tsp, it should not be treated as   operation group here.

Reply:
Correction, as there's operation name conflict in this rlc layer, enableOperationGroup is set as true, therefore we will use the interface name as the operation group name. which should take effect in both RLC and modular layer. So this change is expected.